FieldOpt/CMakeLists.txt

```diff
@@ -62,7 +88,7 @@ if (NOT BUILD_WIC_ONLY)
     # Copy execution scripts
     file(GLOB EXECUTION_SCRIPTS
             "Simulation/execution_scripts/*.sh")
-    file(COPY ${EXECUTION_SCRIPTS} DESTINATION ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/execution_scripts)
+    file(COPY ${EXECUTION_SCRIPTS} DESTINATION ${CMAKE_BINARY_DIR}/execution_scripts)
```

Comment:
Oh, now I remember, it was because the three lines:

set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)

were removed in a previous version of this file, and therefore CMAKE_RUNTIME_OUTPUT_DIRECTORY was not working. Actually, no, they are not equivalent, now that I look at it. I'll fix it back to CMAKE_RUNTIME_OUTPUT_DIRECTORY.